src/components/sections/ProductSpotlight/ProductSpotlight.tsx

```
@@ -4,59 +4,17 @@ import Image from "next/image";
-    size: [38, 39, 40, 41, 42, 43],
-  },
-];
 
 const ProductSpotlight = () => {
-  const [selectedVariant, setSelectedVariant] = useState(productData[0]);
-  const [selectedImage, setSelectedImage] = useState(productData[0].src);
-  const [selectedSize, setSelectedSize] = useState(productData[0].size[0]);
+  if (!PRODUCT_DATA || PRODUCT_DATA.length === 0) {
+    return null;
+  }
+  const [selectedVariant, setSelectedVariant] = useState(PRODUCT_DATA[0]);
```

ESLint

[error] React Hook "useState" is called conditionally. React Hooks must be called in the exact same order in every component render. Did you accidentally call a React Hook after an early return?

(react-hooks/rules-of-hooks)